build.rs

```diff
@@ -114,10 +114,19 @@ mod download {
             );
             println!("url:{}", url);
             let mut downloaded_bytes = Vec::new();
-            let resp = ureq::get(&url).call().unwrap();
-            assert_eq!(resp.status(), 200, "url {} didn't return 200", url);
```

Comment:
I don't remember exactly why this assert is here but maybe it helps with reporting? Can we restore it?